crates/client/src/lib.rs

```diff
@@ -378,6 +379,29 @@ impl<R: RegistryStorage, C: ContentStorage> Client<R, C> {
                 .prove_log_consistency(old_cp.log_root, new_cp.log_root)
                 .await?;
         }
+
+        let operator_records = response.operator;
+        for record in operator_records {
+            let record: ProtoEnvelope<operator::OperatorRecord> = record.try_into()?;
+            let mut operator_info = storage::OperatorInfo::default();
```

Comment:
This should be removed and instead use the operator information loaded previously (there should be one load and one save).